pkg/kubelet/cm/container_manager_linux.go

```diff
@@ -227,7 +229,121 @@ func setupKernelTunables(option KernelTunableBehavior) error {
 	return utilerrors.NewAggregate(errList)
 }
 
+const (
```

Comment:
Move all this logic into a separate interface to make it easy to test. Then inject it into container manager as appropriate.